creating/export-blender-to-aframe/index.md

```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
+Here is some handy boilerplate code to use. make a new  file in your favorite text editor, and copy this into it. I like Sublime Text quite a bit, but any text editor is fine. Name it index.html so that the server can find it. We’ll come back to the code, later.
+
+'''
+<!--.....................................................
+Attribution for 3D model used in this scene: Oculus Rift CV1 by Ti is licensed under CC Attribution-NonCommercial
+.........................................................-->
+
+<!DOCTYPE html>
+   <head>
+    <meta charset="utf-8">
+    <title>Exporting Blender Models from Sketchfab to A-Frame – WebVR Rocks</title>
+    <script src="assets/js/aframe.min.js"></script>
+    <link rel="icon" sizes="192x192" href="https://aframe.io/images/aframe-logo-192.png">
+   </head>
+  <body>
+<!--All of the entities in your scene must be between the <a-scene> tags!-->
+    <a-scene>
+<!--......................................
+It's a good idea to pre-load and cache your assets (stuff in the scene) by using an Asset System. This is an example of an Asset System. Everything in <a-assets> can be used as many times as you want in the scene! You can also use entities inline, without an Asset System: quicker to write, but slower to render. <asset-item> can include geometry, models, and materials
+..........................................-->
+    <a-assets>
+    <a-asset-item id="Oculus-CV1-obj" src="cv1_FF.obj"></a-asset-item>
+    <a-asset-item id="Oculus-CV1-mtl" src="cv1_FF.mtl"></a-asset-item>
+    <a-asset-item id="Oculus-CV1-collada" src="cv1_FF.dae"></a-asset-item>
+    </a-assets>
+
+<!--Sky color can be adjusted-->
+    <a-sky color="#5CAAEB"></a-sky>
+
+<!--Camera has WASD controls by default. In desktop (non VR) mode, click and drag to look around.-->
+    <a-camera position="0, 1.8, 5"></a-camera>
+
+<!--Here are the .obj and .dae models.-->
+      <a-entity obj-model="obj: #Oculus-CV1-obj; mtl: #Oculus-CV1-mtl" position="-5 0 -5"></a-entity>
+      <a-entity collada-model="#Oculus-CV1-collada" position="5 0 -5"></a-entity>
+
+    </a-scene>
+  </body>
+</html>
+'''
+
+## Get npm (if you don’t have it already)
+npm provides an easy way to install packages from the command line, using the Node.js Javascript runtime. 
+If you are on Windows or Mac, installing the latest version of Node will also install npm on your machine.
+
+After installing Node, check that it’s installed properly and check the version of Node and npm. Open command prompt and type:
+
+'''
+node -v 
+nmp -v
+'''
+
+Each of these commands should print a version number. 
+
+## Get http-server
+You can use any local dev server you are comfortable with. Here, I chose http-server, because it’s simple to install with npm, and easy to use.
+To install the server globally, simply enter this on the command line:
+npm install http-server -g
+
+To start the server and open a browser showing your web page:
+http-server [full/path/to/index.html] -o
```

Comment:
you should always invoke `http-server` with `http-server -c-1` to disable caching for local development